backends/advanced/src/advanced_omi_backend/services/obsidian_service.py

```
@@ -0,0 +1,447 @@
+        """
+        try:
+            clean_query = self._clean_text(query)
+            vectors = await generate_openai_embeddings(
+                [clean_query],
+                api_key=self.openai_api_key,
+                base_url=self.openai_base_url,
+                model=self.embedding_model,
+            )
+            query_vector = vectors[0] if vectors else None
+        except Exception as e:
+            logger.exception(f"Embedding failed for search query: {e}")
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)